packages/design-system/src/components/keyboard/index.tsx

```diff
@@ -55,50 +63,41 @@ const CLICKABLE_INPUT_TYPES = [
   'reset',
 ];
 
-const globalRef = createRef();
+const globalRef = createRef<HTMLElement>();
 
 function setGlobalRef() {
   if (!globalRef.current) {
     globalRef.current = document.documentElement;
```

Comment:
Not sure what to do with this error 🤔 
`Cannot assign to 'current' because it is a read-only property.`

Reply:
https://stackoverflow.com/questions/73967855/cannot-assign-to-current-because-it-is-a-read-only-property